### PR TITLE
Run tests also on pull requests

### PR DESCRIPTION
**Pull request by mudler**
As we don't have anymore the chart to test here, there is no need to push the CI image anymore during tests. This allows us to also use ttl.sh and also firing up tests on PRs

Signed-off-by: Ettore Di Giacinto <edigiacinto@suse.com>

### CODE REVIEW

File: .github/workflows/integration-tests.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 concurrency:
   group: integration-tests=full-${{ github.head_ref || github.ref }}-${{ github.repository }}
   cancel-in-progress: true
 
 env:
-   DOCKER_USERNAME: ${{ secrets.QUAY_USERNAME }}
-   DOCKER_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
-   DOCKER_REGISTRY: quay.io
-   REPO: quay.io/costoolkit/os2-ci
+   DOCKER_REGISTRY: ttl.sh
```
Comment (ldevulder) [inline .github/workflows/integration-tests.yaml:10]: I think I will have to do the same for the e2e test, no?

**Review by Itxaka — COMMENTED** [review]
This will duplicate any pr that comes from a branch of the original repo. This only works for fork+pr. I dunno, duplicate stuff with the same sha usually leads to issues... You know I'm not a friend of that 😉

File: .github/workflows/integration-tests.yaml (hunk 1 above)
Comment (mudler) [reply .github/workflows/integration-tests.yaml:10]: they run on master so shouldn't be necessary

**Comment by mudler** [comment]
> This will duplicate any pr that comes from a branch of the original repo. 

You mean that you will run 2 jobs? If there is a nicer way to do with github without going into any complex workflow setups.. I'm all for it.

> This only works for fork+pr. I dunno, duplicate stuff with the same sha usually leads to issues... You know I'm not a friend of that wink

any suggestion?

**Review by Itxaka — APPROVED** [review]

**Comment by mudler** [comment]
@Itxaka thanks for reviewing this!

**Pull request merged**